frontend/src/form/FormEventFragments.types.ts

```diff
@@ -240,6 +240,20 @@ export interface GQLChartWidget extends GQLWidget {
   chart: GQLChart;
 }
 
+export interface GQLLabelWidget extends GQLWidget {
+  label: string;
+  stringValue: string;
+  style: GQLLabelWidgetStyle;
+}
+export interface GQLLabelWidgetStyle {
```

Comment:
Please add an empty line between `GQLLabelWidget` and `GQLLabelWidgetStyle`

Reply:
done